kandy-geo.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "Kotlin plotting DSL for Lets-Plot",
+  "properties": [
+    {
+      "name": "kandyVersion",
+      "value": "0.8.0"
+    },
+    {
+      "name": "kandyVersion-renovate-hint",
+      "value": "update: package=org.jetbrains.kotlinx:kandy-geo"
+    },
+    {
+      "name": "dataframeVersion",
+      "value": "0.15.0"
+    },
+    {
+      "name": "dataframeVersion-renovate-hint",
+      "value": "update: package=org.jetbrains.kotlinx:dataframe-geo"
+    }
+  ],
+  "link": "https://github.com/Kotlin/kandy",
+  "repositories": [
+    "https://repo.osgeo.org/repository/release"
+  ],
+  "dependencies": [
```

Comment:
Why is there no statistics in dependencies?

Reply:
You don't need them when you're working with geo. If you need it, you can simply add them with a usual kandy.

Reply:
Are you suggesting doing like this:
```
%use kandy
%use kandy-geo
```
?

Reply:
Yes.